modules/services/emacs.nix

```diff
@@ -24,18 +24,23 @@ in
         description = "This option specifies the emacs package to use.";
       };
 
+      exec = mkOption {
```

Comment:
I think a command option with the full path and a default of `"${cfg.package}/bin/emacs"` is perhaps a little nicer and more flexible.